packages/core-components/api-report.md

```diff
@@ -333,15 +333,32 @@ export type EmptyStateClassKey = 'root' | 'action' | 'imageContainer';
 // @public (undocumented)
 export type EmptyStateImageClassKey = 'generalImg';
 
-// Warning: (ae-forgotten-export) The symbol "State" needs to be exported by the entry point index.d.ts
+// Warning: (ae-forgotten-export) The symbol "ErrorBoundaryState" needs to be exported by the entry point index.d.ts
 //
 // @public (undocumented)
-export const ErrorBoundary: ComponentClass<ErrorBoundaryProps, State>;
+export class ErrorBoundary extends Component<
+  PropsWithChildren<ErrorBoundaryProps>,
+  ErrorBoundaryState
+> {
+  constructor(props: ErrorBoundaryProps);
+  // (undocumented)
+  componentDidCatch(error: Error, errorInfo: ErrorInfo): void;
+  // (undocumented)
+  static getDerivedStateFromError(error: Error): {
+    error: Error;
+  };
+  // (undocumented)
+  render(): React_2.ReactNode;
+  // (undocumented)
+  resetErrorBoundary(...args: any[]): void;
+}
 
 // @public (undocumented)
 export type ErrorBoundaryProps = {
+  FallbackComponent?: React_2.ComponentType<FallbackProps>;
+  onError?: (error: Error, errorInfo: ErrorInfo) => void;
```

Comment:
The second parameter changed which technically is a breaking change, but I think that's probably OK to ignore here because it was arguably really wrong to begin with.